ionoscloud/resource_nic_test.go

```diff
@@ -164,7 +164,7 @@ resource "ionoscloud_nic" "database_nic" {
   lan = 2
   dhcp = false
   firewall_active = false
-  ips            = [ ionoscloud_ipblock.test_server.ips[0], ionoscloud_ipblock.test_server.ips[1] ]
+  ips = [ ionoscloud_ipblock.test_server.ips[0], ionoscloud_ipblock.test_server.ips[1] ]
```

Comment:
Not in this PR purpose, but here i would also tests (if the field is updatable) if you can empty the ips list or add one more ip

Reply:
good point, will add tests for these